opencensus/metrics/transport.py

```diff
@@ -61,19 +62,23 @@ def func(*aa, **kw):
             except Exception:
                 logger.exception("Error handling metric export")
 
-        super(MetricExporterTask, self).__init__(interval, func, args, kwargs)
+        super(PeriodicMetricTask, self).__init__(interval, func, args, kwargs)
 
 
-def get_exporter_thread(metric_producer, exporter, interval=None):
+def get_exporter_thread(metric_producers, exporter, interval=None):
     """Get a running task that periodically exports metrics.
 
     Get a `PeriodicTask` that periodically calls:
 
-        exporter.export_metrics(metric_producer.get_metrics())
+        export(itertools.chain(*all_gets))
 
-    :type metric_producer:
+        where all_gets is the concatenation of all metrics produced by
```

Comment:
Dedent this to get it to render as text, and consider wrapping the code blocks in backticks. You might want to generate the docs to see how this looks.